Ollama script error (curl: (22) The requested URL returned error: 404)

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

ollama lxc

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/main/ct/ollama.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Ubuntu 24.04

### 📝 Provide a clear and concise description of the issue.

Hi

I'm trying to use the Ollama script on an updated Proxmox 8.4.1 and I get a curl 404 error as below

  🧩  Using Advanced Settings on node nuc
  🖥️  Operating System: ubuntu |   🌟  Version: 24.04
  📦  Container Type: Privileged
  🔐  Root Password: ********
  🆔  Container ID: 102
  🏠  Hostname: ollama
  💾  Disk Size: 90 GB
  🧠  CPU Cores: 4
  🛠️  RAM Size: 12288 MiB
  🌉  Bridge: vmbr0
  📡  IPv4: DHCP
  📡  IPv6: SLAAC/AUTO
  📡  APT-Cacher IP Address: Default
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  📡  Tags: community-script;ai
  🔑  Root SSH Access: yes
  🔍  Verbose Mode: no
  🚀  Creating a Ollama LXC using the above advanced settings
  ✔️   Using local for Template Storage.
  ✔️   Using DATI for Container Storage.
  ✔️   LXC Template List Updated
  ✔️   LXC Template 'ubuntu-24.04-standard_24.04-2_amd64.tar.zst' is ready to use.
  ✔️   Started LXC Container
  ✔️   Customized LXC Container
curl: (22) The requested URL returned error: 404
Remove this Container? <y/N> y
  ✔️   Removed this Container
lxc-attach: 102: ../src/lxc/attach.c: get_attach_context: 406 Connection refused - Failed to get init pid
lxc-attach: 102: ../src/lxc/attach.c: lxc_attach: 1470 Connection refused - Failed to get attach context


If I say N to remove the container I get other error and again the question regarding removing the container.
[ERROR] in line 1285: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVED/raw/branch/main/install/$var_install.sh)" $?

Remove this Container? <y/N> y
/dev/fd/63: line 61: pop_var_context: head of shell_variables not a function context

### 🔄 Steps to reproduce the issue.

Just followed normal script for lxc container

### ❌ Paste the full error output (if available).

  🧩  Using Advanced Settings on node nuc
  🖥️  Operating System: ubuntu |   🌟  Version: 24.04
  📦  Container Type: Privileged
  🔐  Root Password: ********
  🆔  Container ID: 102
  🏠  Hostname: ollama
  💾  Disk Size: 90 GB
  🧠  CPU Cores: 4
  🛠️  RAM Size: 12288 MiB
  🌉  Bridge: vmbr0
  📡  IPv4: DHCP
  📡  IPv6: SLAAC/AUTO
  📡  APT-Cacher IP Address: Default
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  📡  Tags: community-script;ai
  🔑  Root SSH Access: yes
  🔍  Verbose Mode: no
  🚀  Creating a Ollama LXC using the above advanced settings
  ✔️   Using local for Template Storage.
  ✔️   Using DATI for Container Storage.
  ✔️   LXC Template List Updated
  ✔️   LXC Template 'ubuntu-24.04-standard_24.04-2_amd64.tar.zst' is ready to use.
  ✔️   Started LXC Container
  ✔️   Customized LXC Container
curl: (22) The requested URL returned error: 404
Remove this Container? <y/N> y
  ✔️   Removed this Container
lxc-attach: 102: ../src/lxc/attach.c: get_attach_context: 406 Connection refused - Failed to get init pid
lxc-attach: 102: ../src/lxc/attach.c: lxc_attach: 1470 Connection refused - Failed to get attach context

[ERROR] in line 1285: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVED/raw/branch/main/install/$var_install.sh)" $?

Remove this Container? <y/N> y
/dev/fd/63: line 61: pop_var_context: head of shell_variables not a function context

### 🖼️ Additional context (optional).

_No response_

--- a/.github/workflows/push-to-gitea.yaml
+++ b/.github/workflows/push-to-gitea.yaml
@@ -26,7 +26,7 @@ jobs:
           if [ -n "$files_with_github_urls" ]; then
             echo "$files_with_github_urls" | while read file; do
               if [ -f "$file" ]; then
-                sed -i 's|https://raw\.githubusercontent\.com/community-scripts/ProxmoxVE/|https://git.community-scripts.org/community-scripts/ProxmoxVED/raw/branch/|g' "$file"
+                sed -i 's|https://raw\.githubusercontent\.com/community-scripts/ProxmoxVE/|https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/|g' "$file"
               fi
             done         
           else

